packages/ckeditor5-core/src/editor/editor.ts

```diff
@@ -384,22 +384,22 @@ export default abstract class Editor extends /* #__PURE__ */ ObservableMixin() {
 
 			if ( !licenseKey ) {
 				/**
-				 * The `licenseKey` is missing in the editor configuration. If you use premium features,
-				 * please provide your license key. If you do not have a key yet, please contact us at
-				 * [https://ckeditor.com/contact/](https://ckeditor.com/contact/) or order a trial at
-				 * [https://orders.ckeditor.com/trial/premium-features](https://orders.ckeditor.com/trial/premium-features).
+				 * The `licenseKey` property is missing in the editor configuration.
 				 *
-				 * If you do not use premium features, add the `'GPL'` license key instead.
+				 * 	* If you are using the editor in a commercial setup, please provide your license key.
+				 * 	* If you still need to acquire a key, please contact us or start a free trial.
+				 * 	* If you are using the editor under a GPL license or another license from our Open Source Initiative ,
+				 * 	  use the 'GPL' license key instead.
 				 *
```

Comment:
The section above misses the links from the GDoc.

Reply:
Whoops, missed that, good point

Reply:
Fixed https://github.com/ckeditor/ckeditor5/pull/17400/commits/1c3bb05e44e76009a0bf7106feaa02bf17a91360